builder/azure/arm/azure_client.go

```diff
@@ -149,7 +161,13 @@ func NewAzureClient(subscriptionID, resourceGroupName, storageAccountName string
 		return nil, err
 	}
 
-	storageClient, err := storage.NewBasicClient(storageAccountName, *accountKeys.Key1)
+	storageClient, err := storage.NewClient(
+		storageAccountName,
+		*(*accountKeys.Keys)[0].Value,
```

Comment:
this seems like a strange API change... did they really change from two properties into an array?